packages/gestalt-codemods/generic-codemods/modifyProp.js

```diff
@@ -0,0 +1,87 @@
+// @flow strict
+
+/**
+ * CODEMOD to MODIFY (DEPRECATE or  RENAME) GESTALT COMPONENT PROPS
+ * Ex. <Box size="" /> to <Box />
+ * Ex. <Box size="" /> to <Box renamedSize="" />
+ * Ex. <Dropdown.Item size="" /> to <Dropdown.Item renamedSize="" />
+ *
+ *
+ * OPTIONS:
+ * --action: 'deprecate' removes prop matching previousPropName, 'rename' replaces prop name matching previousPropName with nextPropName
```

Comment:
This should really be "remove" — "deprecate" indicates marking a feature as obsolete and to be removed, but we're actually removing it here

Though also: is this necessary? Could removal also be communicated by passing `--nextPropName=null`?

Reply:
good point. DONE